services/QuillLMS/app/controllers/api/v1/rule_feedback_histories_controller.rb

```diff
@@ -24,4 +24,12 @@ def prompt_health
     report = PromptFeedbackHistory.run(**options)
     render json: report
   end
+
+  def activity_health
+    raise ArgumentError unless params.include?('activity_id')
+
+    options = params.permit(:activity_id).to_h.symbolize_keys
```

Comment:
I think `permit` is unnecessary because of the guard clause 2 lines above.

Reply:
FYI.  ActionController::Parameters (i.e. `params`) [prevents](https://github.com/rails/rails/pull/28734) calls `to_h` on  without a `permit`.